Automattic/jetpack
# Action Bar: remove module and mark package as abandoned

Fixes #34811

## Proposed changes:

The module is no longer developed, and does not need to be in Jetpack anymore.

### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* Go to Jetpack > Settings > Modules
* You should no longer see an Action bar module.
